### PR TITLE
[DA-599] Refine INTERESTED criteria and adjust synthetic calendar

### PR DESCRIPTION
**Pull request by eweitz**
This refines INTERESTED criteria to include individuals who have signed up but not yet consented.  It also adjusts the range of dates used in a calendar for synthetic data generation.  This is needed to ensure Metrics API 1 and Metrics API 2 counts align in our test environment.

### CODE REVIEW

File: rest-api/dao/participant_counts_over_time_service.py [hunk 1]
```diff
@@ -181,7 +181,7 @@ def get_enrollment_status_sql(self, filters_sql_ps, filters_sql_p):
             FROM calendar c2
             LEFT OUTER JOIN
             (SELECT COUNT(*) cnt,
-                (CASE WHEN enrollment_status = 1 THEN
+                (CASE WHEN (enrollment_status = 1 OR enrollment_status IS NULL) THEN
```
Comment (danqrodney) [inline rest-api/dao/participant_counts_over_time_service.py]: As discussed offline, this needs to be updated to use table_prefix:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/dao/participant_counts_over_time_service.py#L124

**Comment by eweitz** [comment]
I implemented tests, but as discussed on Slack yesterday creating a participant sans participant summary (and no enrollment status) isn't yielding a new INTERESTED count as expected.  I'm working on that.

**Comment by eweitz** [comment]
That last tweak to check for NULL email got my new test passing, @danqrodney.  Thanks for that.

File: rest-api/test/unit_test/api_test/participant_counts_over_time_api_test.py [hunk 2]
```diff
@@ -539,6 +546,35 @@ def test_get_counts_excluding_withdrawn_participants(self):
...
+
+    qs = ''.join(qs.split())  # Remove all whitespace
+
+    response = self.send_get('ParticipantCountsOverTime', query_string=qs)
+
+    total_count_day_1 = response[0]['metrics']['INTERESTED']
+    total_count_day_2 = response[1]['metrics']['INTERESTED']
+
+    self.assertEquals(total_count_day_1, 0)
+    self.assertEquals(total_count_day_2, 3)
```
Comment (danqrodney) [inline rest-api/test/unit_test/api_test/participant_counts_over_time_api_test.py]: I thought this was supposed to be 2?

Comment (danqrodney) [reply rest-api/test/unit_test/api_test/participant_counts_over_time_api_test.py]: Never mind, should be 3. LGTM!

**Pull request merged**